src/main/java/racingCar/controller/RacingCarController.java

```diff
@@ -0,0 +1,41 @@
+package racingCar.controller;
+
+import racingCar.domain.CarGroup;
+import racingCar.util.NumberGenerator;
+import racingCar.view.InputView;
+import racingCar.view.OutputView;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+public class RacingCarController {
+
+    public void run(NumberGenerator numberGenerator) {
+        CarGroup carGroup = repeat(this::makeCars);
+        Integer tryCount = repeat(InputView::readTryCount);
+        moveCarsAndView(numberGenerator, tryCount, carGroup);
+        OutputView.printWinners(carGroup.findWinners());
+    }
+
+    private CarGroup makeCars() {
+        List<String> carNames = repeat(InputView::readCarNames);
+        CarGroup carGroup = new CarGroup(carNames);
+        return carGroup;
```

Comment:
변수로 별다른 작업이 없으면 인라인 해주면 좋을것 같아요
```suggestion
        return new CarGroup(carNames);
```